docker/deploy.sh

```diff
@@ -6,12 +6,9 @@ sudo apt-get update && sudo apt-get install -y python-pip && sudo pip install aw
 
 # Get login token and execute login
 $(aws ecr get-login --no-include-email --region $AWS_REGION)
-echo -e $GITLAB_PRIVATE_KEY > .ssh/id_rsa
-chmod 0500 .ssh/id_rsa
 
-echo "Building latest image with solver...";
-docker-compose build --build-arg use_solver=1 stablex
-docker tag stablex $REGISTRY_URI:$TRAVIS_BRANCH
+echo "Tagging latest image with solver...";
+docker tag stablex-rinkeby $REGISTRY_URI:$TRAVIS_BRANCH
```

Comment:
Why `stablex-rinkeby`? Would it make sense to make the environments as close as possible so their differents are abstracted away just with env var (like the provider url)

Reply:
If you look inside the docker compose the only difference is the .env file. `docker-compose up` doesn't allow adding an env file which is why we added another target. But we can revisit this if you have a better suggestion.

Reply:
docker-compose run has `-e` so you could pass an env var. Would this help to abstract away the differences?

Reply:
Yes, unfortunately `docker-compose run` doesn't allow a daemon mode though, so it would be blocking (or would have to be called with & at the end). This could work however, since locally we don't usually run the daemon mode.